experiments/04_object_storage_crash/download_file.yml

```diff
@@ -3,7 +3,7 @@
   hosts: controller
   roles:
     - config
-  sudo: True
+  become: yes
```

Comment:
Keep the PRs small by fixing just one problem.  Next time, simply open another PR to fix deprecation warnings.